src/functions/advanced-functions/event.php

```diff
@@ -94,6 +94,10 @@
 	 * }
 	 *
 	 * @return int|bool ID of the event that was created. False if insert failed.
+	 * @category Events
+	 *
+	 * @version  TBD
+	 * @since    TBD Updated docblock to indicate future deprecation of function.
```

Comment:
There is no need for `@version`, but I would move this to the top of the Docblock, right after the free-text "area"
```suggestion
	 * @since    3.0.0
```